core/src/main/scala/com/salesforce/op/evaluators/OpBinaryClassificationEvaluator.scala

```diff
@@ -112,10 +114,29 @@ private[op] class OpBinaryClassificationEvaluator
       val falsePositiveRateByThreshold = sparkMLMetrics.roc().collect().map(_._1).slice(1, thresholds.length + 1)
       val aUROC = sparkMLMetrics.areaUnderROC()
       val aUPR = sparkMLMetrics.areaUnderPR()
+
+      // Solutions can be checked with any system of equations solver. For Wolfram Alpha, input:
+      // inv {{(1-P), -P, 0, 0}, {(1-R), 0, 0, -R}, {0, 1-"Fpr", -"Fpr", 0}, {1,1,1,1}} * {0, 0, 0, "Count"}
+      val (tupPos, tupNeg) = thresholds.zipWithIndex.map{ case (_, i) => {
```

Comment:
Ooo, good point - if any of the spark quantities are 0 that will cause problems. I can rewrite things so that there's less division, but I may need to put in a regularization param for some of them

Reply:
yeah we dont want to fail a flow for a divide by zero here

Reply:
Removed!